.drone.yml

```
@@ -0,0 +1,46 @@
+---
+
+name: default
+kind: pipeline
+type: kubernetes
+node_selector:
+  CPUs: 8
+
+steps:
+
+  - name: get version -- branch
+    image: ubuntu:latest
+    commands:
+      - echo $(cat VERSION)-$(echo $DRONE_COMMIT_BRANCH | sed 's/[/_-]//g')-$DRONE_BUILD_NUMBER > .tags
+      - echo $(cat .tags)
+    when:
+      branch:
+        exclude:
+          - master
+      event:
+        - push
+
+  - name: release server image -- branch
+    image: plugins/gcr
+    settings:
+      repo: cyrus-containers/thermompnn
+      tags:
+        - ${DRONE_COMMIT_BRANCH}
+      debug: true
+      dockerfile: Dockerfile
+      json_key:
+        from_secret: dockerconfigjson
+    when:
+      event: push
+
+  - name: release server image -- tag
+    image: plugins/gcr
+    settings:
+      repo: cyrus-containers/thermompnn
+        - ${DRONE_TAG}
```

missing a key in the yaml file
```suggestion
        tags:
        - ${DRONE_TAG}
```